prime-angular-frontend/openshift.dockerfile

```diff
@@ -36,15 +36,14 @@ COPY nginx.template.conf /etc/nginx/nginx.template.conf
 COPY entrypoint.sh /
 RUN echo "Populating environment..." && \
     (eval "echo \"$(cat /usr/src/app/src/environments/environment.prod.template.ts )\"" ) > /usr/src/app/src/environments/environment.prod.ts
-RUN cat /usr/src/app/src/environments/environment.prod.ts && \ 
-    npm install @angular/cli@8.3.24  -g --silent && \ 
-    npm install && \ 
-    npm audit fix --only=prod && \ 
-    ng build --prod && \ 
+RUN cat /usr/src/app/src/environments/environment.prod.ts && \
+    npm install @angular/cli  -g --silent && \
+    npm install && \
+    npm audit fix && \
+    ng build --prod && \
     echo "NPM packages installed..."
 
-# RUN npm audit fix --only=prod && \ 
-#     npm install @angular-devkit/build-angular@0.803.24 \ 
+# RUN npm audit fix --only=prod && \
```

Comment:
Same here

Reply:
@poggdogg do we need this commented out code in the Dockerfile?